sched/definitions.go

```diff
@@ -181,18 +178,20 @@ func makeThriftJobFromDomainJob(domainJob *Job) (*schedthrift.Job, error) {
 		}
 		jobId := domainJob.Id
 		taskId := domainTask.TaskID
-		thriftTask := schedthrift.TaskDefinition{Command: &cmd, JobId: &jobId, TaskId: &taskId}
+		requestorTag := domainTask.RequestorTag
+		thriftTask := schedthrift.TaskDefinition{Command: &cmd, JobId: &jobId, TaskId: &taskId, RequestorTag: &requestorTag}
 		thriftTasks = append(thriftTasks, &thriftTask)
 	}
 
 	prio := int32(domainJob.Def.Priority)
 	thriftJobDefinition := schedthrift.JobDefinition{
-		JobType:   &(*domainJob).Def.JobType,
-		Tasks:     thriftTasks,
-		Priority:  &prio,
-		Tag:       &(domainJob).Def.Tag,
-		Basis:     &(domainJob).Def.Basis,
-		Requestor: &(domainJob).Def.Requestor,
+		JobType:      &(*domainJob).Def.JobType,
+		Tasks:        thriftTasks,
+		Priority:     &prio,
+		Tag:          &(domainJob).Def.Tag,
+		Basis:        &(domainJob).Def.Basis,
+		Requestor:    &(domainJob).Def.Requestor,
+		RequestorTag: &(domainJob).Def.RequestorTag,
```

Comment:
Tag, Requestor, and RequestorTag?

See scootapi/scoot.thrift for tag and requestor descriptions - I don't think we're supposed to have a third RequestorTag entry.

Reply:
My understanding was that RequestorTag would be provided by the requestor specifically for the purpose of tracing a job through its lifecycle. Requestor and Tag seem to have orthogonal purposes wrt this goal. Were you thinking of repurposing Tag?

Reply:
That was the original purpose of Tag, it's kind of implied that a tag would be from one requestor. I don't really care about the naming, I just don't think we need both Tag and RequestorTag unless you have another distinction in mind.

Reply:
Got it - replacing with Tag